src/MooLite/plugins/EquipmentExporter/EquipmentExporterPlugin.ts

```diff
@@ -37,11 +37,11 @@ export class EquipmentExporterPlugin extends MooLitePlugin {
         return equippedItems;
     }
 
-    getFoodSummary(): Record<ActionTypeHrid, CharacterConsumable[]> | null {
+    getFoodSummary(): Record<ActionTypeHrid, (CharacterConsumable | null)[]> | null {
         return this._game.inventory.getEquippedFood();
     }
 
-    getDrinkSummary(): Record<ActionTypeHrid, CharacterConsumable[]> | null {
+    getDrinkSummary(): Record<ActionTypeHrid, (CharacterConsumable | null)[]> | null {
```

Comment:
The final `| null` can never happen as we instantiate it in the class. Could you remove them?